### PR TITLE
test_show_intf_xcvr.py: Added command "show interface transceiver lpmode"  

### PR DESCRIPTION
**Pull request by vesuresh537**
ading show command "show interface transceiver lpmode"

### Description of PR

Summary:
Fixes # (issue)

### Type of change
Added New cli  "show interface transceiver lpmode" in  tech_support_cmds.py

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)  imorovement
- [ ] Test case(new/improvement).  imorovement

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305

#### What is the motivation for this PR?
Added New cli  "show interface transceiver lpmode" in  tech_support_cmds.py
#### How did you verify/test it?
Verified on new to topology

### CODE REVIEW

File: tests/platform_tests/sfp/test_show_intf_xcvr.py
Comment (bpar9) [inline tests/platform_tests/sfp/test_show_intf_xcvr.py:72]: similar to other procs in this script, will be good to have this proc in util.py in the parent directory

File: tests/platform_tests/sfp/test_show_intf_xcvr.py
Comment (bpar9) [inline tests/platform_tests/sfp/test_show_intf_xcvr.py:67]: also need to for if the interface is not in the xcvr skip list and then check for the valid lpmode output

**Review by bpar9 — CHANGES_REQUESTED** [review]

File: tests/platform_tests/sfp/test_show_intf_xcvr.py
Comment (vesuresh537) [reply tests/platform_tests/sfp/test_show_intf_xcvr.py]: Done

**Comment by nareshkmr** [comment]
@prgeor could you please review this code?

**Comment by madhupalu** [comment]
@yxieca @prgeor - It's been waiting long time, btw Prince George is code owner for this repo.  Can you guys approve this PR or assign to right owner. 
Thanks

**Comment by yxieca** [comment]
> @yxieca @prgeor - It's been waiting long time, btw Prince George is code owner for this repo. Can you guys approve this PR or assign to right owner. Thanks

@vesuresh537, @madhupalu please address the checker failures.

**Review by prgeor — APPROVED** [review]

**Review by bpar9 — APPROVED** [review]

**Comment by nareshkmr** [comment]
@yxieca Could you please look at the PR and approve if nothing else is to be addressed?

**Pull request merged**
